content/docs/2.0/scalers/azure-event-hub.md

```diff
@@ -34,7 +34,16 @@ triggers:
 
 ### Authentication Parameters
 
-Not supported yet.
+You can authenticate by using connection string authentication.
+
+**Connection String Authentication:**
+
+- `connection` - Connection string for the Azure Service Bus Namespace.
+  
+  The following formats are supported.
+  
+  - With **SharedAccessKey** - `Endpoint=sb://<sb>.servicebus.windows.net/;SharedAccessKeyName=<key name>;SharedAccessKey=<key value>`
```

Comment:
```suggestion
  - With **SharedAccessKey** - `Endpoint=sb://<sb>.servicebus.windows.net/;SharedAccessKeyName=<key name>;SharedAccessKey=<key value>;EntityPath=<hub-name>`
```

Reply:
This applies for all versions 2.0.x => 2.8.x.

Reply:
Done